build.sbt

```diff
@@ -214,7 +214,7 @@ lazy val V = new {
   def scala2Versions = nonDeprecatedScala2Versions ++ deprecatedScala2Versions
 
   // Scala 3
-  def nonDeprecatedScala3Versions = Seq(scala3, "0.27.0-RC1")
+  def nonDeprecatedScala3Versions = Seq(scala3, "3.0.0-M1", "0.27.0-RC1")
```

Comment:
Just because I don't fully remember where we ended up on this. How long will we keep 0.27.0-RC1 around as non-deprecated?

Reply:
Probably under the end of the course at EPFL.